### PR TITLE
Update General section to match osu file format v14

### PR DESCRIPTION
**Pull request by stkjoe**
- Reordered section contents in General to follow order of appearance in the latest osu file format
- Updated countdown description
- Added SkinPreference, CountdownOffset

### CODE REVIEW

File: wiki/osu!_File_Formats/Osu_(file_format)/en.md [hunk 1]
```diff
@@ -60,22 +60,30 @@ LetterboxInBreaks (Boolean) specifies whether the letterbox appears during break
...
-WidescreenStoryboard (Boolean) specifies whether or not the storyboard should be widescreen.
-
-`WidescreenStoryboard: 0`
-
 StoryFireInFront (Boolean) specifies whether or not display the storyboard in front of combo fire.
 
 `StoryFireInFront: 0`
 
-SpecialStyle (Boolean) specifies whether or not use the special `N+1` style for osu!mania.
+SkinPreference (String) specified the preferred skin to use during gameplay.
```
Comment (cl8n) [inline wiki/osu!_File_Formats/Osu_(file_format)/en.md]:
```diff
- specified
+ specifies
```

Comment (stkjoe) [reply wiki/osu!_File_Formats/Osu_(file_format)/en.md]: Oops my bad. How do I add the typo fix in? New to Github

File: wiki/osu!_File_Formats/Osu_(file_format)/en.md [hunk 2]
```diff
@@ -60,22 +60,30 @@ LetterboxInBreaks (Boolean) specifies whether the letterbox appears during break
...
+SkinPreference (String) specifies the preferred skin to use during gameplay.
 
-`SpecialStyle: 0`
+`SkinPreference:Default`
 
 EpilepsyWarning (Boolean) specifies whether or not show a 'This beatmap contains scenes with rapidly flashing colours...' warning at the beginning of the beatmap.
 
 `EpilepsyWarning: 0`
 
+CountdownOffset (Boolean) specifies how many beats earlier the countdown starts.
```
Comment (TPGPL) [inline wiki/osu!_File_Formats/Osu_(file_format)/en.md]: "how many beats earlier"
"Boolean" 
is this one correct?

**Comment by cl8n** [comment]
@stkjoe are you around to apply TPGPL's review? this is ready to merge after that

**Pull request closed without merging**